Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
I'm sorry, but I don't understand this statement at all. The same applies to the commit message that belongs to the commit that introduces these changes. (Note that I'm not opposing these changes, just suggesting they could be applied to all such instances.)